app/controllers/api/v3/logins_controller.rb

```diff
@@ -7,21 +7,27 @@ class Api::V3::LoginsController < APIController
   before_action :validate_login_payload, only: %i[create]
 
   def login_user
-    authentication = PhoneNumberAuthentication.find_by(phone_number: login_params[:phone_number])
-    errors = errors_in_user_login(authentication)
+    result = PhoneNumberAuthentication::Authenticate.call(otp: login_params[:otp],
```

Comment:
I'm wondering if this should be `PhoneNumberAuthentication.authenticate()` and hide the internal class?